quick_tests.py

```diff
@@ -261,10 +262,13 @@ def print_filter(d_filter):
 
     print "Clear Router Diagnostics"
     print "========================"
-    transceiver.clear_router_diagnostic_counters(0, 0)
-    router_diagnostics = transceiver.get_router_diagnostics(0, 0)
-    print router_diagnostics.registers
-    print ""
+    transceiver.clear_router_diagnostic_counters(
+        0, 0, counter_ids=[router_diagnostics._REGISTERS.LOC_PP.value,
```

Comment:
shouldnt the router_diagnostics be a enum inside spinnman, to a point where its public assessable (as a front end user shouldnt need to access a private enum to get/clear a specfic router counter.